initial upload of v1 for videos and newletters as new resource type

Rogzy-DB opened the pull request:
very early proposal of data strcuture with 2 exemple. 

Please update to final data structure so Edo can start linking back and front and we can push . I have a long list of YTB channel and documantary ready to be added once we agree on the strcuture.

Rogzy-DB committed:
Proposal dat structur v0.1 Video

PaoloFoti commented:
@Rogzy-DB newsletter folder template is not present in this commit despite the name of the branch. However, the proposal for video structure regarding youtube channel and movies-documentary seems good to me

BitPreto committed:
upload of v1 for videos and newletters as new resource type (#958)

* add id to course questions (#901)

* Update vi.md (#767)

* Proofreading - wallet-sentinel-vi (#769)

* Update vi.md

* Update vi.md

* Update vi.md (#776)

* Proofreading eco102 - vi (#777)

* Proofreading eco102 - vi

* Update vi.md

* Update vi.md

* Proofreading eco201 - vi (#784)

* Update vi.md

* Update vi.md

* Update vi.md

* add missing proofreading metadata

* eco201, add vi proofreading contribution

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

* Proofreading min201 - vi (#820)

* Update vi.md

* Update vi.md

* Update es.md (#783)

* Update es.md (#805)

* BTC204-rus proofreading (#849)

* Section 1.1

* Intro

* Intro

* Section 1

* Section 1

* Section 1

* Section 1

* Section 2

* Section 2

* Section 2.2

* Section 2.3

* Section 2.3

* Section 2.3

* Section 2.3

* Section 2.3

* Section3

* Section 3

* Section 3

* Section 3

* Section 3

* Section 3

* Section 3

* Section 4

* Section 4

* Section 4

* Section 4

* Section 5

* Section 5

* Section 5.3

Working on Section 5.3 Joinmarket

All Section before 5.3 have been proofread

* Section 5.3

S5.3 Wabisabi

* Section 5.4

Whirlpool

* Section 5.4 

Section 5.4 TX0

* Section 5.4

Section 5.4
How to properly manage your postmix?

* Sectoin 5.4

Toxic change

* Section 5.5

Section 5.5

* Section 5.6

* Section 5.6

Entropy

* Section 5.6

* Section 5.6

Fix the equaiton to get the end of the section to be displayed

The end of the Section 5.6

* Section 6.1

Section 6.1

* Section 6.2

Section 6.2
Stonewall

* Section 6.2

Section 6.2
Stonewall x2

* Section 6.3

Section 6.3
Ricochets

* Section 6.3

Section 6.3 
Ricochets

* Section 6.3

Ricochets

* Section 6.4

Seciton 6.4
CoinSwap

* Section 6.4

Section 6.4
Atomic Signatures

* Section 6.4 

Section 6.4
Atomic Swap

* Section 6.5

Section 6.5
Dandelion

* Section 6.5

Section 6.5
P2p
Tor

* Section 6.6

BIP47

* Section 6.6

Section 6.6

Elliptic-Curve Diffie-Hellman

* Section 6.6

Section 6.6
ECDH
Notification Transaction

* Section 6.6

* Section 6.6

Section 6.6
The privacy model of bip47

* Section 6.6

Section 6.6

Construction of the Notification Transaction

* Section 6.6

Section 6.6
Notification Transaction: Concrete Study
XOR

* Section 6.6

Section 6.6
Receiving the Notification Transaction

* Section 6.6

add the image to the end of the 
Receiving the Notification Transaction

* Section 6.6

* Section 6.6

Section 6.6
Receiving the BIP47 Payment and Deriving the Private Key

* Section 6.6

Section 6.6
Refunding the BIP47 Payment

* Section 6.7

Section 6.7
Why not move the notification?

* Section 6.7

Section 6.7
Naive derivation of a Silent Payments public key

* Section 6.7

Section 6.7
How to create multiple outputs?

* Section 6.7

Section 6.7
How to avoid address reuse?

* Section 6.7

Section 6.7
Tweaking the public keys in inputs
Separating Spending and Scanning Keys

* Section 6.7

Section 6.7
Tweaking the public keys in inputs

* Section 6.7

Section 6.7
Separating Spending and Scanning Keys

* Section 6.7

Section 6.7
How to Construct a Silent Payments Address?

* Section 6.9

Conclusion

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* Test proofreading (#904)

* fix key properties

* allow to catch error with missing key

* update proofreading metadata

* Update es.md (#853)

* Profreading [ITA]  (#857)

* Update it.md

up to 281

* Update it.md

up to 797

* Update it.md

* Update it.md

up to 178

* Update it.md

up to 382

* Update it.md

up to 430

* Update it.md

up to 786

* Update it.md

up to 1063

* Update it.md

up tp 1723

* Update it.md

* Update it.md

* New Bitcoin events (#834)

* Nicehashx Event 2024

* Fix events of  bitcoin++ 2025

* Sat'n'facts conference 2025

* Add mining 302 Course in the event page, closes #896

* Add run-for-hal to event page (closes #897)

* Add Liquid Summit to event page (closes #897)

* Add Satoshi Spritz to event page (closes #897)

* Add thunderstorm  to event page (closes #897)

* Add Lugano  builders to the event page (fixes #897)

* Add Lugano B-Cert Exam English Version

* Add Lugano B-Cert Exam Eng Version (fixes #898)

* Fix title Lugano B-Cert Exam Eng Version (fixes #898)

* Fix date Bitcoin Amsterdam (fixes #899)

* Fix PlanB Lugano Forum Date, fixes #900

* Tuto TAPSIGNER (#828)

* docs

* txt + orthographe

* img

* Update fr.md

* Update 03.webp

* UUID + metadata

* UUID + metadata (#830)

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* Re-formating BTC401 (#835)

* Tutorial SATSCARD (#836)

* docs

* Theoretical txt + intro

* Update tutorial.yml

* txt + orthographe

* img

* Update fr.md

* rgn

* Update tutorial.yml

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* Tutorial Ledger Flex (#870)

* docs

* Update fr.md

* txt ok + orthographe

* Update fr.md

* Update fr.md

* img

* rgn

* Update fr.md

* Update tutorial.yml

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* Add: New Replay Conference Mining Disrupt 2024 (#889)

* Add: New Replay Conference Mining Disrupt 2024

* Fix:Language Replay  Mining Disrupt 2024

* Add:New PBN tags to Replay Mining Disrupt 2024. Closes #888

* Update conference.yml

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* btc101, fix teacher

* fix chapter id

* fix chapter id

* Bcert remove websiteurl

* ₿-Cert's Events PlanB Lugano Forum Eng and It  (#923)

* ₿-Cert's Events PlanB Lugano Forum Eng and It, fixes #920

* Fix timezone bug in Lugano Satellite Events #921

* Delete BOBFERENCE 2025 (Canceled) #924

* Game B show Lugano 2024, fixes #926

* fix naming guidelines for Plan ₿

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

* Lugano PlanB forum side events (#902)

* initial commit for spazio21@PlanB Forum event

* initial commit for SSL forum edition 2024 event

* added thumbnail for ssl-2024-lugano + added telegram group link

* updated Spazio21 description to specify that contents are in Italian

* updated SSL Forum Edition thumbnail

* updated URLs for both events and Spazio21 thumbnail

* fix naming guideline for Plan ₿

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* fix naming guideline

* upload of v1 for videos and newletters as new resource type

newsletter structure is provided with two examples, please note that there are no info about atlas21's example. testing purposes only.

YT channels and movies-documentaries data structure is the same of rogzy's proposal #893

* updated newsletter example

* update new resources type

* update dummy files for new resource types

* videos, add ids

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>
Co-authored-by: Henry-Da0 <169113863+Henry-Da0@users.noreply.github.com>
Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>
Co-authored-by: Oscar Guevara <100944058+OscarG673@users.noreply.github.com>
Co-authored-by: Reshetenko <100995119+Reshetenko@users.noreply.github.com>
Co-authored-by: TinF0ilHat1 <127325867+TinF0ilHat1@users.noreply.github.com>
Co-authored-by: jramos0 <56746361+jramos0@users.noreply.github.com>
Co-authored-by: Loïc Morel <137194052+LoicPandul@users.noreply.github.com>
Co-authored-by: trigger67 <10185784+trigger67@users.noreply.github.com>
Co-authored-by: Marco Giorgetti <marco@marcogiorgetti.com>
